docs/3.1. Discovery - Registered Operation.md

```diff
@@ -8,86 +8,98 @@ This document describes usage of NMOS APIs for discovery in cases where where a
 
 ### DNS-SD Advertisement
 
-The preferred method of Registration API advertisement is via unicast DNS-SD advertisement of the type \_nmos-register.\_tcp. Registration APIs MUST additionally be capable of producing an mDNS advertisement. This MAY be disabled via a user-configurable method.
+The preferred method of Registration API advertisement is via unicast DNS-SD advertisement of the type `_nmos-register._tcp`. Registration APIs MUST additionally be capable of producing an mDNS advertisement. This MAY be disabled via a user-configurable method.
 
-Registration APIs supporting versions 'v1.2' and below MUST additionally be capable of producing an mDNS advertisement of type \_nmos-registration.\_tcp. This MAY be disabled via a user-configurable method.
-*note: RFC6763 Section 7.2 specifies that the maximum service name length for an mDNS advertisement is 16 characters when including the leading underscore, but "_nmos-registration" is 18 characters.*
+Registration APIs supporting versions `v1.2` and below MUST additionally be capable of producing an mDNS advertisement of type `_nmos-registration._tcp`. This MAY be disabled via a user-configurable method.
```

Comment:
Does the MAY here apply to the server or the client/user? I.e. Is it valid for a server not to support this user configurability?